packages/webapp/pages/posts/[id].tsx

```diff
@@ -121,7 +121,7 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
   if (isPostLoadingOrFetching || isFallback || !isFetched) {
     return (
       <>
-        {seoComponent}
+        {post?.title?.length && seoComponent}
```

Comment:
This doesn't introduce latency loading?
A bit worried about other sharing platforms but let's give it a go